include/revng/CliftTransforms/Passes.td

```diff
@@ -11,6 +11,13 @@ class Clift_Pass<string arg, string operation> : Pass<arg, operation> {
   let dependentDialects = ["clift::CliftDialect"];
 }
 
+def CliftTrivialReturnElimination : Clift_Pass<"trivial-return-elimination",
```

Comment:
We've got no unit test for this. Unless it's in another commit that I haven't seen yet.

It's easy to come up with simple but non-trivial cases to unit test, both for testing that the optimization expectedly succeeds in some cases, and expectedly fails in others.